x-pack/plugins/apm/public/components/routing/templates/settings_template.tsx

```diff
@@ -5,90 +5,108 @@
  * 2.0.
  */
 
-import { EuiPage, EuiPageBody, EuiPageSideBar, EuiSideNav } from '@elastic/eui';
+import { EuiPageHeaderProps } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import React, { ReactNode, useState } from 'react';
+import React from 'react';
+import { History } from 'history';
 import { useHistory } from 'react-router-dom';
+import { CoreStart } from 'kibana/public';
+import { ApmMainTemplate } from './apm_main_template';
 import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
 import { getAPMHref } from '../../shared/Links/apm/APMLink';
 
-export function SettingsTemplate({ children }: { children: ReactNode }) {
+type Tab = NonNullable<EuiPageHeaderProps['tabs']>[0] & {
+  key:
+    | 'agent-configurations'
+    | 'anomaly-detection'
+    | 'apm-indices'
+    | 'customize-ui';
+  hidden?: boolean;
+};
+
+interface Props {
+  children: React.ReactNode;
+  selectedTab: Tab['key'];
+}
+
+export function SettingsTemplate({ children, selectedTab }: Props) {
   const { core } = useApmPluginContext();
   const history = useHistory();
-  const { basePath } = core.http;
-  const canAccessML = !!core.application.capabilities.ml?.canAccessML;
-  const { search, pathname } = history.location;
+  const tabs = getTabs({ history, core, selectedTab });
 
-  const [isSideNavOpenOnMobile, setisSideNavOpenOnMobile] = useState(false);
+  return (
+    <ApmMainTemplate
+      pageHeader={{
+        tabs,
+        pageTitle: i18n.translate('xpack.apm.settings.title', {
+          defaultMessage: 'APM Settings',
```

Comment:
```suggestion
          defaultMessage: 'Settings',
```

IMO the added APM is redundant in the title, it should be clear from the navigation and breadcrumbs what area these settings related to.